mw-webapp/src/converter/userConverter.ts

```diff
@@ -0,0 +1,14 @@
+import {querySnapshot} from "src/converter/querySnapshot";
+import {User as UserDTO} from "src/model/firebaseCollection/User";
```

Comment:
should we rename User to UserDO in firebaseCollection folder (and other models)

Reply:
why? won't this be confusing, since there are certain collections on firebase?

Reply:
rename it please)

Reply:
Done